go.mod

```diff
@@ -3,9 +3,11 @@ module github.com/utilitywarehouse/kube-service-mirror
 go 1.14
 
 require (
+	github.com/google/go-cmp v0.3.0
 	github.com/hashicorp/go-hclog v0.11.0
 	github.com/stretchr/testify v1.4.0
 	k8s.io/api v0.17.4
 	k8s.io/apimachinery v0.17.4
 	k8s.io/client-go v0.17.4
+	k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
```

Comment:
can't this be same version as above, or a newer one, but not come from a commit?

Reply:
This was added automatically, probably by `go mod tidy` magic. Do you want to declare a version manually?

Reply:
I think that it should match the above `k8s.io` submodules

Reply:
That package doesn't really have releases https://github.com/kubernetes/utils/releases